tests/test_collector.py

```diff
@@ -33,77 +27,67 @@ def test_combine_metric(self, DogStatsd):
             'namespace.test2',
         )
 
-        metrics.init(namespace='default_namespace')
-        collector = metrics.Collector(namespace='namespace', prefix='test1')
+        with mock.patch('mbq.metrics._namespace', 'default_namespace'):
+            collector = metrics.Collector(namespace='namespace', prefix='test1')
+            self.assertEqual(
+                collector._combine_metric('test2'),
+                'namespace.test1.test2',
+            )
+            collector = metrics.Collector(prefix='test1')
+            self.assertEqual(
+                collector._combine_metric('test2'),
+                'default_namespace.test1.test2',
+            )
+
+    @mock.patch('mbq.metrics._constant_tags', {'a': 1, 'b': 1})
+    def test_combine_tags(self, _statsd):
```

Comment:
These tests were really complicated. I think they're now easier to understand and demonstrate the functionality.